sdk/cli/test/test_arv-get.rb

```diff
@@ -1,251 +1,192 @@
 require 'minitest/autorun'
-require 'digest/md5'
+require 'json'
+require 'yaml'
 
+# Black box tests for 'arv get' command.
 class TestArvGet < Minitest::Test
-  def setup
-    begin
-      Dir.mkdir './tmp'
-    rescue Errno::EEXIST
-    end
-    @@foo_manifest_locator ||= `echo -n foo | ./bin/arv-put --filename foo --no-progress -`.strip
-    @@baz_locator ||= `echo -n baz | ./bin/arv-put --as-raw --no-progress -`.strip
-    @@multilevel_manifest_locator ||= `echo ./foo/bar #{@@baz_locator} 0:3:baz | ./bin/arv-put --as-raw --no-progress -`.strip
-  end
-
-  def test_no_args
-    out, err = capture_subprocess_io do
-      assert_arv_get false
-    end
-    assert_equal '', out
-    assert_match /^usage:/, err
-  end
-
-  def test_help
-    out, err = capture_subprocess_io do
-      assert_arv_get '-h'
-    end
-    $stderr.write err
-    assert_equal '', err
-    assert_match /^usage:/, out
-  end
-
-  def test_file_to_dev_stdout
-    test_file_to_stdout('/dev/stdout')
-  end
-
-  def test_file_to_stdout(specify_stdout_as='-')
-    out, err = capture_subprocess_io do
-      assert_arv_get @@foo_manifest_locator + '/foo', specify_stdout_as
-    end
-    assert_equal '', err
-    assert_equal 'foo', out
-  end
-
-  def test_file_to_file
-    remove_tmp_foo
-    out, err = capture_subprocess_io do
-      assert_arv_get @@foo_manifest_locator + '/foo', 'tmp/foo'
-    end
-    assert_equal '', err
-    assert_equal '', out
-    assert_equal 'foo', IO.read('tmp/foo')
-  end
-
-  def test_file_to_file_no_overwrite_file
-    File.open './tmp/foo', 'wb' do |f|
-      f.write 'baz'
-    end
-    out, err = capture_subprocess_io do
-      assert_arv_get false, @@foo_manifest_locator + '/foo', 'tmp/foo'
-    end
-    assert_match /Local file tmp\/foo already exists/, err
-    assert_equal '', out
-    assert_equal 'baz', IO.read('tmp/foo')
-  end
+  # UUID for an Arvados object that does not exist
+  NON_EXISTANT_OBJECT_UUID = "qr1hi-tpzed-p9yk1lihjsgwew0"
```

Comment:
existant &rarr; existent

use "zzzzz" instead of "qr1hi" to avoid conflating "uuid does not exist" and "uuid is not in our namespace and does not exist" (currently those cases are handled identically, so I'm just thinking of future federation features and human readers)